CMakeLists.txt

```diff
@@ -183,12 +161,10 @@ add_subdirectory (manual)
 
 file(GLOB_RECURSE extrafiles
     RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
-    cmake/* fonts/* stylesheet/* xsl/* docbook/*)
+    cmake/* fonts/* xsl/* docbook/*)
 add_to_dist(${extrafiles})
 
-configure_file(COPYING ${DATADIR_BUILD}/${PROJECT_NAME}/COPYING COPYONLY)
-configure_file(COPYING-DOCS ${DATADIR_BUILD}/${PROJECT_NAME}/COPYING-DOCS COPYONLY)
-install(FILES COPYING COPYING-DOCS DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME})
+install(FILES COPYING COPYING-DOCS DESTINATION "share/doc/${PROJECT_NAME}")
```

Comment:
Your choice of "doc" as subdirectory is nice, though this change will require coordination with @derekatkins to update the nightly build scripts. Please do it in a separate commit to unblock the rest of the changes.

On the other hand I'm uncomfortable with hardcoding  'share' rather than using  CMAKE_INSTALL_DATADIR. I understand you want to drop module GNUInstallDirs to avoid having to set a (programming) language, but this limits configurability and may cause certain distros to no longer be able to install properly.

If you don't want GNUInstallDirs, you should define a cache variable intitially set to "share" in the top level CMakeLists.txt  file and use that everywhere (including in the build steps that output in (a subdirectory) of builddir/share). That allows packagers to change the destination path as needed.